### DISCUSSION TITLE
Welcome to EarTensifier Discussions!

### DISCUSSION THREAD

**Discussion by 3eif** [3eif/ear-tensifier#68] [Announcements]
hi lol

**Comment by LucaPierino** [comment]
Do you have any update on future of this project ?

Ciao Luca

**Reply by 3eif** [reply]
The project is still alive and maintained, it just won't be open source any time soon.

**Reply by LucaPierino** [reply]
Hi,
I had tried the bot some time ago, but at that point it was no longer working properly with YouTube videos, so I stopped using it.

About your comment: when you say the project won’t be open source any time soon, do you mean that it will remain closed source permanently, or that you might consider making it open source later on?

Thanks for the clarification!
Luca
